pdc_client/runner.py

```diff
@@ -72,8 +72,13 @@ def load_plugins(self):
             raise TypeError('Server must be specified')
         config = pdc_client.read_config_file(server)
         plugins = DEFAULT_PLUGINS
-        if config and config[CONFIG_PLUGINS_KEY_NAME]:
-            plugins = config[CONFIG_PLUGINS_KEY_NAME]
```

Comment:
Looks like change condition to `if config and config.get(CONFIG_PLUGINS_KEY_NAME):` could work as you expected.